test/test_rdoc_ruby_lex.rb

```diff
@@ -591,96 +591,39 @@ def test_class_tokenize_string
     assert_equal expected, tokens
   end
 
-  def test_class_tokenize_string_with_escape
+  def test_class_tokenize_postfix_if_after_escaped_newline
```

Comment:
`test_class_tokenize_string_with_escape` is needless test case? It seems regression.